core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -170,8 +171,10 @@ case class ModelInsights
       val params = e.modelParameters.filterKeys(!excludedParams.contains(_))
       Seq("name" -> e.modelName, "uid" -> e.modelUID, "modelType" -> e.modelType) ++ params
     }).flatten.sortBy(_._1)
-    val table = Table(name = name, columns = Seq("Model Param", "Value"), rows = validationResults)
-    table.prettyString()
+    if (validationResults.nonEmpty) {
+      val table = Table(name = name, columns = Seq("Model Param", "Value"), rows = validationResults)
+      table.prettyString()
+    } else "No validation results found"
```

Comment:
I would rather change the return type of this function to `Seq[String]` and return `Seq.empty` if `validationResults` are empty.